fairmq/FairMQDevice.cxx

```diff
@@ -511,6 +515,18 @@ void FairMQDevice::RunWrapper()
         {
             while (CheckCurrentState(RUNNING) && ConditionalRun())
             {
+              using TimeScale = std::chrono::microseconds;
+              static const auto reftime = std::chrono::system_clock::now();
```

Comment:
No, it's independent of the type of the clock. Anything which discourages the use of `system_clock` in this case?

Reply:
Although it would not break your code: `system_clock` is not required to be monotonic, so timespan could be negative. `steady_clock` is required to be monotonic, which fits more the expectation one would have from a clock in this algorithm imho.